policy/modules/system/userdomain.if

```diff
@@ -3667,6 +3670,25 @@ interface(`userdom_relabelfrom_user_runtime_dirs',`
 	allow $1 user_runtime_t:dir relabelfrom;
 ')
 
+########################################
+## <summary>
+##     write user runtime socket files
+## </summary>
+## <param name="domain">
+##     <summary>
+##     Domain allowed access.
+##     </summary>
+## </param>
+#
+interface(`userdom_write_all_user_runtime_named_sockets',`
+	gen_require(`
+		attribute user_runtime_content_type;
+	')
+
+	allow $1 user_runtime_content_type:dir list_dir_perms;
```

Comment:
Is it possible to downgrade this access to `search_dir_perms`?

Reply:
OK I'll change that for now. We can merge less privs and then decide later whether to add more.